### PR TITLE
[AWS 4.13] Add UPI CI jobs

### PR DESCRIPTION
**Pull request by yunjiang29**
Add AWS UPI CI jobs for OCP 4.13

### CODE REVIEW

**Comment by yunjiang29** [comment]
@jianlinliu @liangxia please review this PR, thanks.

4.10 - 4.12 will be added in another PR.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml [hunk 1]
```diff
@@ -749,6 +749,30 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-aws-sc2s-ipi-disconnected-private
+- as: aws-upi-p3-f28
+  cron: 0 19 */28 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      E2E_RUN_TAGS: '@aws-ipi and @network-ovnkubernetes and not @fips'
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:758]: It should be `aws-upi`

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml [hunk 2]
```diff
@@ -749,6 +749,30 @@ tests:
...
+      TAG_VERSION: '@4.13'
+      TEST_FILTERS: ~ChkUpgrade&;~NonPreRelease&;Serial;Disruptive;~ConnectedOnly&;~HyperShiftMGMT&
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:760]: `~Serial&;~Disruptive&`

Comment (yunjiang29) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml]: fixed.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml (hunk 1 above)
Comment (yunjiang29) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml]: fixed.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml [hunk 3]
```diff
@@ -749,6 +749,30 @@ tests:
...
+    workflow: cucushift-installer-rehearse-aws-upi
+- as: aws-upi-p3-f28-destructive
+  cron: 0 19 */28 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      E2E_RUN_TAGS: '@aws-upi and @network-ovnkubernetes and not @fips and @destructive'
+      TAG_VERSION: '@4.13'
+      TEST_FILTERS: ~ChkUpgrade&;~NonPreRelease&;~Serial&;~Disruptive&;~ConnectedOnly&;~HyperShiftMGMT&
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:772]: Here should be `Serial;Disruptive`

Comment (yunjiang29) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml]: fixed

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml [hunk 4]
```diff
@@ -749,6 +749,30 @@ tests:
...
+    workflow: cucushift-installer-rehearse-aws-upi
+- as: aws-upi-p3-f28-destructive
+  cron: 0 19 */28 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      E2E_RUN_TAGS: '@aws-upi and @network-ovnkubernetes and not @fips and @destructive'
+      TAG_VERSION: '@4.13'
+      TEST_FILTERS: ~ChkUpgrade&;~NonPreRelease&;Serial;Disruptive;~ConnectedOnly&;~HyperShiftMGMT&
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:772]: I think aws-upi is a connected cluster, so here (and above) should use `~DisconnectedOnly&`

Comment (yunjiang29) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml]: fixed

Comment (jianlinliu) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml]: Oh, I misunderstood `~`, ignore my comment.

**Comment by jianlinliu** [comment]
To wait for all the jobs are completed and destroyed.

/hold

**Comment by liangxia** [comment]
> To wait for all the jobs are completed and destroyed.
> 
> /hold

The e2e/upgrade can be terminated, but the destroy (post) is always executed.

**Comment by jianlinliu** [comment]
> The e2e/upgrade can be terminated, but the destroy (post) is always executed.

Good to know.

/unhold

**Pull request merged**